README

```diff
@@ -18,3 +18,4 @@ usage : tty-clock [-iuvsScbtrahDBxn] [-C [0-7]] [-f format] [-d delay] [-a nsdel
     -B            Enable blinking colon
     -d delay      Set the delay between two redraws of the clock. Default 1s.
     -a nsdelay    Additional delay between two redraws in nanoseconds. Default 0ns.
+    -p [1-9]      Position of clock in terminal: 1-9 starting top-left and ending bottom-right.
```

Comment:
"starting top-left and ending bottom-right"... how do we get to "bottom left"? :)

i guess we want to say something like "starting clockwise from the top-left" or something like that.

should we make 0 be "center" and remove the other flag? ;)